modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -828,30 +827,26 @@ private boolean hasTextField() throws MBFormatException {
      *
      * @return The font to use for the label
      */
+
     public List<String> getTextFont() {
         String[] fonts;
-        if (layout.get("text-font") instanceof JSONObject) {
-            JSONArray stops = (JSONArray) ((JSONObject) layout.get("text-font")).get("stops");
-            fonts = ((JSONArray)((JSONArray) stops.get(0)).get(1)).get(0).toString().split(",");
-        } else {
+        if (!(layout.get("text-font") instanceof JSONObject)) {
```

Comment:
Bit of a nitpick, but since we already have an if-else, using:

```
if (layout.get("text-font") instanceof JSONObject) {
    return null
} else {
    ...
}
```

would be a bit clearer.